native/cocos/audio/android/PcmAudioService.cpp

```diff
@@ -113,7 +125,7 @@ bool PcmAudioService::init(AudioMixerController *controller, int numChannels, in
     const SLInterfaceID ids[] = {
         SL_IID_PLAY,
         SL_IID_VOLUME,
-        SL_IID_ANDROIDSIMPLEBUFFERQUEUE,
+        IDD_BUFFER_QUEUE,
```

Comment:
It's better not to modify the logic for Android. I think AndroidSimpleBufferQueue is different with BufferQueue.
To make sure you don't break anything if you have to modify it, you need to refer  to the source code of Android's opensles implementation.